plasma_framework/contracts/src/vaults/Vault.sol

```diff
@@ -8,6 +8,10 @@ contract Vault is Operated {
     PlasmaFramework framework;
     bytes32[16] zeroHashes;
 
+    address private _currentDepositVerifier;
```

Comment:
why keeping all of this as private fields?
If we keep those fields public, then the compiler will generate getters for us - less code to maintain, deploy.

Reply:
Does `public` means any other contract can change this immediately on the Vault contract?
I want to keep code that is allowed to modify these values close together.

Reply:
https://solidity.readthedocs.io/en/v0.5.10/contracts.html#getter-functions

Reply:
It will not allow anyone to change the value.

Reply:
I think it will though? The child contract can access and rewrite the value right?